etna/generic_pages/tests/test_blocks.py

```diff
@@ -5,22 +5,22 @@
 
```

Comment:
Maybe `test_models.py` would be a better name for this module?

Reply:
Done